### ISSUE TITLE
On Windows - "cec-client not installed" error

### ISSUE THREAD

**Issue by moryoav** [damoclark/node-red-contrib-cec#9]
On Windows 10, the CEC client is installed properly, from the CMD (both as regular user and as an administrator) I can run "cec-client" and it works fine. In Node-Red though I get the "cec-client not installed" error. I've added the folder "C:\Program Files (x86)\Pulse-Eight\USB-CEC Adapter\x64" to the path in Windows.

Edit: I changed the line commandExists("cec-client") to commandExists("dir") and it's also not working. I think commandExists is faulty on Windows.

Edit: I manually added the folder C:\Program Files (x86)\Pulse-Eight\USB-CEC Adapter\x64 to the PM2 dump. That solved it. It was a path issue.

**Comment by lizaoreo** [comment]
@moryoav Sorry to dig up something this old, but I'm having the same issue and this is the only place I've seen the issue discussed.  Can you elaborate on what you mean by PM2 dump?  Haven't been able to figure that out.

**Comment by moryoav** [comment]
Sorry, for the life of me I can't remember what this was, it's been 5 years since... I'm not using any of those things anymore. I don't have this installed anymore to even check the folder. Sorry.

**Comment by lizaoreo** [comment]
@moryoav No worries, thank you for responding.  I think late last night I got it working.  I did not have PM2 installed (didn't realize it was a whole sperate thing and my searching related to the dump file so everything I found just assumed I already had it installed).  After I started looking for what it is, I figured out it was a separate package and installed it and used it to start Node-Red, stuff just started working.
